Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnUpdate.hs

```diff
@@ -306,6 +312,11 @@ onUpdate ValidatedRideAssignedReq {..} = do
             safetyCheckStatus = Nothing,
             ..
           }
+    checkIsDriverBirthDay driverBirthDate = do
+      let (_, birthMonth, birthDay) = toGregorian $ utctDay driverBirthDate
+      currentTime <- getCurrentTime
```

Comment:
Let's take local time

Reply:
but the driver's dob in driver license is in utc

Reply:
Fixed